core/arch/opnd.h

```diff
@@ -2843,7 +2843,31 @@ DR_API
 bool
```

Comment:
> Adds tests for above by duplicating relevant api.ir evex cases using a compresseable

Spelling: "compresseable"

Also s/above/the above/ or sthg

Reply:
Fixed.